packages/office-addin-debugging/src/start.ts

```diff
@@ -52,9 +61,32 @@ export async function isPackagerRunning(statusUrl: string): Promise<boolean> {
 export function parseAppType(text: string): AppType | undefined {
     switch (text) {
         case "desktop":
+        case "macos":
+        case "win32":
             return AppType.Desktop;
         case "web":
             return AppType.Web;
+        case "ios":
+        case "android":
+            throw new Error(`Platform type ${text} not currently supported for debugging`);
```

Comment:
I think should parse AppType even for ios and android (as Desktop). Instead of having unsupported platforms fail the parse, you should check the enum value where appropriate.